aci/resource_aci_fvsubnet.go

```diff
@@ -132,20 +132,18 @@ func getRemoteSubnet(client *client.Client, dn string) (*models.Subnet, error) {
 
 func setSubnetAttributes(fvSubnet *models.Subnet, d *schema.ResourceData) (*schema.ResourceData, error) {
 	dn := d.Id()
-	d.SetId(fvSubnet.DistinguishedName)
+	ip := d.Get("ip").(string)
+	d.Set("ip", ip)
 	d.Set("description", fvSubnet.Description)
 	if dn != fvSubnet.DistinguishedName {
-		d.Set("parent_dn", "")
+		d.SetId(dn)
```

Comment:
This does not seems correct. Why are we setting it to dn if it is not identical?